core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -45,7 +45,7 @@ class AnalyzerManager(
       case Some(moduleId) =>
         f(req, moduleId)
       case None =>
-        sauron ! EnsimeServerError(s"Couldn't find the project for ${fileInfo.file}")
+        sauron forward req
```

Comment:
I'll still need to make a few changes in the case statement of `TypeCheckFilesReq`.
I will add some test. :)